<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Console</class>
 <widget class="QMainWindow" name="Console">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Reborn</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background:transparent;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">

QGroupBox {
	color: white;
	font-family: Verdana;
	font-size: 11px;
}

QRadioButton {
	font-family: Verdana;
	color: white;
	font-size: 11px;
}
QRadioButton::indicator::unchecked {
	height: 18px;
	width: 18px;
	border-image: url(:/iGEM2012GUI/Resources/ui images/unchecked.png);
}
QRadioButton::indicator:checked {
	height: 18px;
	width: 18px;
	border-image: url(:/iGEM2012GUI/Resources/ui images/checked.png);
}

QLabel {
	color: white;
	font-family: Verdana;
	font-style: bold;
	font-size: 11px;
}

QLabel#label_5 {
	background-image: url(:/iGEM2012GUI/Resources/ui images/radio-background.png);
}

QWidget#centralwidget {	
	border-image: url(:/iGEM2012GUI/Resources/console-bg.png);
}

QTextEdit {	
	border-image: url(:/iGEM2012GUI/Resources/screen-bg.png);
	color: white;
	padding: 10px;
}

QPushButton#min, QPushButton#close {
	border-image: url(:/iGEM2012GUI/Resources/ui images/round_button_normal.png);
}
QPushButton#min:hover {
	border-image: url(:/iGEM2012GUI/Resources/ui images/round_button_min.png);
}
QPushButton#close:hover {
	border-image: url(:/iGEM2012GUI/Resources/ui images/round_button_close_hover.png);
}

QPushButton {
	border-image: url(:/iGEM2012GUI/Resources/ui images/button_normal.png);
	color: white;
}
QPushButton:hover {
	border-image: url(:/iGEM2012GUI/Resources/ui images/button_hover.png);
}

QPushButton:pressed {
	border-image: url(:/iGEM2012GUI/Resources/ui images/button_pushed.png);
}
QPushButton:disabled {
	border-image: url(:/iGEM2012GUI/Resources/ui images/button_inactive.png);
	color: gray;
}



QDoubleSpinBox, QSpinBox {	
	border-image: url(:/iGEM2012GUI/Resources/ui images/stepper_background.png);
	color: white;
	padding-left: 4px;
	padding-right: 15px;
	height: 43px;
}

QDoubleSpinBox::up-button, QSpinBox::up-button {
    subcontrol-origin: border;
    subcontrol-position: top right; 
     
    border-image: url(:/iGEM2012GUI/Resources/ui images/stepper_background_button_up.png);
	width: 30px;
}
QDoubleSpinBox::up-button:hover, QSpinBox::up-button:hover {
    border-image: url(:/iGEM2012GUI/Resources/ui images/stepper_background_button_up_hover.png);
}
QDoubleSpinBox::up-button:pressed, QSpinBox::up-button:pressed {
    border-image: url(:/iGEM2012GUI/Resources/ui images/stepper_background_button_up_pressed.png);
}

QDoubleSpinBox::down-button, QSpinBox::down-button {
    subcontrol-origin: border;
    subcontrol-position: bottom right; 
     
    border-image: url(:/iGEM2012GUI/Resources/ui images/stepper_background_button_down.png);
	width: 30px;
}
QDoubleSpinBox::down-button:hover, QSpinBox::down-button:hover {
    border-image: url(:/iGEM2012GUI/Resources/ui images/stepper_background_button_down_hover.png);
}
QDoubleSpinBox::down-button:pressed, QSpinBox::down-button:pressed {
    border-image: url(:/iGEM2012GUI/Resources/ui images/stepper_background_button_down_pressed.png);
}


QSlider:horizontal {
	border-image: url(:/iGEM2012GUI/Resources/ui images/horizontal-slider-background.png);
	height: 25px;
	padding-left: 8px;
	padding-right: 8px;
}
QSlider::groove:horizontal {
	border-image: url(:/iGEM2012GUI/Resources/ui images/horizontal-slider-field-background.png);
	height: 25px;
}
QSlider::handle:horizontal {
	border-image: url(:/iGEM2012GUI/Resources/ui images/horizontal-slider-handle-background.png);
	height: 25px;
	width: 23px;
}
QSlider::sub-page:horizontal {
	background-image: url(:/iGEM2012GUI/Resources/ui images/horizontal-slider-left-background.png);
	height: 25px;
	margin-left: 3px;
}

QSlider:vertical {
	border-image: url(:/iGEM2012GUI/Resources/ui images/slider_background.png);
	width: 19px;
}

QSlider::groove:vertical {
	background: transparent;
}

QSlider::handle:vertical {
    height: 54px;
	width: 19px;
	border-image: url(:/iGEM2012GUI/Resources/ui images/slider_button.png);
}

QCheckBox::indicator {
    width: 143px;
    height: 44px;
}


QCheckBox::indicator:unchecked{
	image: url(:/iGEM2012GUI/Resources/ui images/off.png);
}
QCheckBox::indicator:checked{
	image: url(:/iGEM2012GUI/Resources/ui images/on.png);
}</string>
   </property>
   <widget class="QPushButton" name="goto1">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>470</y>
      <width>99</width>
      <height>43</height>
     </rect>
    </property>
    <property name="text">
     <string>Map</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goto4">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>340</y>
      <width>99</width>
      <height>43</height>
     </rect>
    </property>
    <property name="text">
     <string>Sandbox</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goto3">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>220</y>
      <width>99</width>
      <height>43</height>
     </rect>
    </property>
    <property name="text">
     <string>Report</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>81</width>
      <height>43</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>81</width>
      <height>43</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>200</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>81</width>
      <height>43</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>81</width>
      <height>43</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generations</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Population</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Survivals</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Outputs</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="defaultButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Default</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rebornButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Let's REBORN!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>0</y>
      <width>76</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="min">
    <property name="geometry">
     <rect>
      <x>946</x>
      <y>7</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>984</x>
      <y>7</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ftp">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>110</y>
      <width>99</width>
      <height>43</height>
     </rect>
    </property>
    <property name="text">
     <string>FTP setting</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>691</width>
      <height>531</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'XHei SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>335</y>
      <width>171</width>
      <height>85</height>
     </rect>
    </property>
    <property name="title">
     <string>Build Method</string>
    </property>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>45</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>promoters and genes</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Operons</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="igem2012gui.qrc"/>
 </resources>
 <connections/>
</ui>
